app/dashboard/agenda/page.tsx

```
@@ -1,7 +1,148 @@
-      <h1 className="text-3xl">Agenda</h1>
+    <div className="min-h-screen bg-blue-50 flex justify-center items-center">
+      <div className="w-11/12 max-w-4xl bg-white rounded-lg shadow-lg p-6">
+        <div className="flex justify-between items-center mb-4">
+          {/* Informações do Usuário */}
+          <div className="flex items-center space-x-4">
+            <img
+              src="/dashboard/img/avatar-cservice-male-500-500.png"
+              width={40}
+              height={40}
+              alt="User Image"
+              className="rounded-full"
```

Biome (1.9.4)

[error] Avoid the words "image", "picture", or "photo" in img element alt text.

Screen readers announce img elements as "images", so it is not necessary to redeclare this in alternative text.

(lint/a11y/noRedundantAlt)